src/actions/askQuestions.js

```diff
@@ -89,6 +89,12 @@ module.exports = async input => {
       message: 'Will you be using TypeScript? (Only affects linting)',
       default: false,
     },
+    {
+      type: 'confirm',
+      name: 'sentry',
+      message: 'Do you want to use Sentry for this project?',
```

Comment:
```suggestion
      message: 'Do you want to add Sentry to this project?',
```
Heel klein semantisch dingetje, omdat je echt een project gaat aanmaken en koppelen

Reply:
Goeie, aangepast.